src/main/resources/assets/mekanism/lang/ru_ru.lang

```diff
@@ -439,13 +449,13 @@ gui.start=Запуск
 gui.stop=Стоп
 gui.config=Настройка
 gui.teleport=Телепорт
-gui.eject=Извлечение
+gui.eject=Выброс
 gui.input=Вход
 gui.strictInput=Строгий вход
 gui.slots=Слоты
 gui.state=Состояние
-gui.on=Вкл.
-gui.off=Выкл.
+gui.on=Да
+gui.off=Нет
```

Comment:
Why change these to yes/no?

Reply:
Same meanings and fits any tiny button on GUI (old ones does not!).

Reply:
(for example "filter: on/off" can be clearly understand in form "filter [active]: yes/no")

Reply:
Hmmm I see. My issue with that is we can't guarantee it will always make sense as the english does not mean the same

Reply:
Make a buttons with dynamic fit to text size.